tests/features/joinup_subscription/subscriptions_manage.feature

```diff
@@ -2,12 +2,110 @@
 Feature: User subscription settings
   As a user I must be able to set and view my subscription settings.
 
-  Scenario Outline: Change the notification frequency of my digests
+  Background:
     Given user:
       | Username   | Auric Goldfinger  |
       | Password   | oddjob            |
       | E-mail     | auric@example.com |
-    And collection:
+
+  Scenario: Check access to the subscription management pages
+    Given user:
+      | Username   | Chanelle Testa    |
+      | E-mail     | chate@example.com |
+
+    # No access for anonymous users.
+    Given I am an anonymous user
+    When I go to the subscription settings form of "Auric Goldfinger"
+    Then I should see the error message "Access denied. You must sign in to view this page."
+    When I go to the subscription dashboard of "Auric Goldfinger"
```

Comment:
Same here. Why not letting the user click-trip to the page. More natural.

Reply:
Same reason as above, when logged in as a user that doesn't have access it is not possible to click through to the page.